Jellyfin.Data/Enums/ImageType.cs

```diff
@@ -8,66 +8,66 @@ public enum ImageType
         /// <summary>
         /// The primary.
         /// </summary>
-        Primary = 0,
```

Comment:
Why did you remove all of the enum values?

Reply:
Afaik, enums are translated to int32 as in Java in C#, so I thought it was useless keeping them and thought it was a good change while I was at the cleanup process :)

However, I'm checking and all the enums in the codebase have their values specified, so maybe we want to have this reverted for consistency. Or apply this change throughout the code base (not for this PR definitely, but I can make it)

Thoughts? @jellyfin/server

Reply:
Specified values is better IMO

Reply:
Makes sense, hadn't thought about it @barronpm 

Adressed!